mx-tester.yml

```diff
@@ -10,9 +11,22 @@ up:
 
 run:
   - yarn test:integration
+
 down:
   finally:
     - docker stop mjolnir-test-reverse-proxy || true
+
+modules:
+  - name: mjolnir
+    build:
+      - cp -r synapse_antispam $MX_TEST_MODULE_DIR
+    install:
```

Comment:
This install step doesn't do anything and should be deleted. mx-tester already takes care of installing the module, this makes it hang.